packages/terra-cell-grid/src/terra-dev-site/test/cell-grid/CellGrid.test.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import CellGrid from '../../../index';
+
+const CellGridTest = () => (
```

Comment:
Should we add tests to account for different divider styles?

Reply:
Added tests.